docs/pages/guides/expo-sdk-explained.md

```diff
@@ -0,0 +1,74 @@
+
+### Versions of the SDK and release schedule
+
+Expo operates on a quarterly SDK release schedule, which allows for meaningful changes that have been thoroughly tested to be included in each SDK, and also provides some predictability for Expo developers. Since we want you to be able to easily work on apps running older SDK versions, the Expo Go app supports usually the last 4 SDK versions. If you're further behind than that, you can always install older versions of Expo Go via `expo-cli` by running `expo client:install:[ios | android]` from your project directory (for iOS, this command only works for simulators, not physical devices).
+
+To upgrade your Expo SDK version, just run `expo upgrade` in your project directory, and `expo-cli` will take care of the rest.
+
+### React Native version
+
+Each Expo SDK includes support for a particular version of React Native. It is usually up-to-date with the most recent _stable_ release (this does mean we will wait after a new release to see if there are any breaking changes or widely impacting bugs that we'd like to see fixed before we upgrade). When you upgrade your Expo SDK with `expo upgrade`, Expo automatically handles upgrading your version of React Native as well.
+
+In the bare workflow, you are [not as tied to the Expo SDK](#significance-of-the-expo-sdk-in), so you can upgrade React Native independent of your current SDK version (this will impact how you can use Expo Go, as detailed further on in this guide.
+
+### What is `expo install`?
+
+Because projects running in Expo Go rely on the native code that is bundled into the SDK, it is important that the version of a package you have in your `package.json` is compatible with the native code it's using. To simplify this, you can use `expo install <some-expo-package>` and the proper version of that package, based on your SDK version, will be installed. In other words- `expo install` installs the appropriate JavaScript for a package based on the native code your project is using (which is indicated by your Expo SDK version).
+
+## Significance of the Expo SDK in:
+
+### Expo Go
+
+Expo Go is where your Expo SDK version is most important. Whether you are in the bare workflow or the managed workflow, if you use Expo Go for development, you should be aware of your Expo SDK version and use `expo install` when you install dependencies.
+
+We version the native code in Expo Go so that you can run projects on different SDK versions easily, but this means you need to tell Expo Go which version of native code to run. This is usually done via the `expo` package in your `package.json` (unless you override it with the `sdkVersion` property in `app.json`), which points to a particular SDK version (e.g. "^41.0.0").
+
+### Non-Expo development
```

Comment:
Bare workflow and React Native?

Reply:
still quite common to use Expo Go in bare

Reply:
yeah the wording is tough here so maybe we can change the structure? something about using native projects directly or not